examples/axes_grid1/demo_axes_rgb.py

```diff
@@ -59,11 +59,8 @@ def demo_rgb2():
     ax_b.imshow(im_b)
 
     for ax in fig.axes:
-        ax.tick_params(axis='both', direction='in')
+        ax.tick_params(axis='both', direction='in', color='w')
```

Comment:
```suggestion
        ax.tick_params(direction='in', color='w')
```
Both is the default.